src/linear_feedback_controller_ros.cpp

```diff
@@ -53,60 +52,32 @@ CallbackReturn LinearFeedbackControllerRos::on_init() {
                  "Issues initializing the introspection.");
     return CallbackReturn::FAILURE;
   }
+  if (parameters_.chainable_controller.command_interfaces.size() !=
+      lfc_.get_robot_model()->get_moving_joint_names().size()) {
+    RCLCPP_ERROR(get_node()->get_logger(),
+                 "Parameter chainable_controller.command_interfaces do not "
+                 "have the correct size.");
+    return CallbackReturn::FAILURE;
+  }
```

Comment:
I'd suggest to add more context information to help debugging/understand the problem inside the logs, especially for error related logs:

```suggestion
  if (parameters_.chainable_controller.command_interfaces.size() !=
      lfc_.get_robot_model()->get_moving_joint_names().size()) {
    RCLCPP_ERROR(get_node()->get_logger(),
                 "Size mismatch between robot model and parameters. "
                 "Robot description have %zu moving joints declared in it "
                 "while chainable_controller.command_interfaces contains %zu",
                 lfc_.get_robot_model()->get_moving_joint_names().size(),
                 parameters_.chainable_controller.command_interfaces.size());
    return CallbackReturn::FAILURE;
  }
```